src/chart/resize/updateDataMarks/drawBars.js

```diff
@@ -60,7 +60,7 @@ export default function drawBars(marks) {
         bars.each(function(d) {
             let mark = select(this.parentNode.parentNode).datum();
             d.tooltip = mark.tooltip;
-            d.arrange = mark.split ? mark.arrange : null;
+            d.arrange = mark.split && mark.arrange ? mark.arrange : mark.split ? 'grouped' : null;
```

Comment:
This lost me a bit - what's this doing?

Reply:
Defining whether and how the bars are grouped (stacked, side-by-side, nested).  I noticed the y-axis bar behavior wasn't matching the x-axis behavior and repeated the code here.